catalog/transformer/catalog-transformer-video-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,7 +13,13 @@
  **/
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
-    <bean id="transformer" class="ddf.catalog.transformer.input.video.VideoInputTransformer"/>
+    <bean id="transformer" class="ddf.catalog.transformer.input.video.VideoInputTransformer">
+        <argument ref="genericMetacardType"/>
+    </bean>
+
+    <reference id="genericMetacardType" interface="ddf.catalog.data.MetacardType"
```

Comment:
This is adding a feature dependency on the video-input-transformer to require the tika-input-transformer be installed first.  Either the metacardType needs to be moved into it's own feature or video-input-transformer needs a prereq on tika-input-transformer.